docs/MAINTENANCE.md

```diff
@@ -0,0 +1,57 @@
+# Maintenance
+
+This document describes updating the [examples](../examples) in this repository to use the latest Cypress version.
+
+## Examples
+
+There are two groups of examples which are used to test and demonstrate the use of [cypress-io/github-action](https://github.com/cypress-io/github-action):
+
+1. The [examples](../examples) directory contains examples of the use of Cypress (Current) [Configuration](https://docs.cypress.io/guides/references/configuration) which applies to Cypress 10 and later.
+
+2. The [examples/v9](../examples/v9) directory contains examples which are set up to use Cypress 9.7.0 which is the last version using [Legacy Configuration](https://docs.cypress.io/guides/references/legacy-configuration) which applies to Cypress 9 and below.
```

Comment:
```suggestion
2. The [examples/v9](../examples/v9) directory contains examples which are set up to use Cypress `9.7.0` which is the last version using [Legacy Configuration](https://docs.cypress.io/guides/references/legacy-configuration) which applies to Cypress 9 and below.
```

Reply:
@AtofStryker 

Unfortunately you were too late with your suggestion and @mjhenkes had already merged. I don't think it is worth a new PR to make the change, but I will bear it in mind if the file needs editing later for some other reason.

Reply:
I applied the suggestion here: https://github.com/cypress-io/github-action/pull/779/commits/9ca965cadc67714f1f7812a449ea48bcf1be5795

Before i merged!